src/components/auth/LoginForm.tsx

```diff
@@ -38,14 +38,22 @@ const LoginForm = () => {
 		try {
 			const res = await dispatch(login(data)).unwrap();
 			saveAccessToken(res.data);
-			showSuccessMessage(res.message);
-			navigate('/');
+
+			if (
+				res.message ===
+				'Email sent for verification. Please check your inbox and enter the OTP to complete the authentication process.'
+			) {
```

Comment:
Please use another comparison but not res.message

Reply:
Corrected it.